docs/recce-cloud/share-recce-session-securely.md

```diff
@@ -17,25 +17,55 @@ Recce Cloud allows you to share your Recce validation results with full context,
 
 When you're ready to share your lineage exploration, query results, or validation checklist, simply click **Share** in Recce.
 
-The first time you do this, you'll need to connect your local Recce to Recce Cloud. This requires signing in and setting up an API token. Once connected, Recce Cloud will host your state file securely, allowing you to share a link that others can open in their browsers.
+The first time you do this, you'll need to connect your local Recce to Recce Cloud. This requires signing in and setting the connection of local Recce. Once connected, Recce Cloud will host your state file securely, allowing you to share a link that others can open in their browsers.
 
 ### 1. Enable Recce Sharing
 
-To start sharing, launch Recce server and click the **Enable Sharing** button.
+To start sharing, launch Recce server and click the **Use Recce Cloud** button if you are not connecting Recce with Recce Cloud before.
 
-![Recce Server](../assets/images/recce-cloud/recce-server-enable-sharing-fs8.png){: .shadow}
+![Recce Server](../assets/images/recce-cloud/recce-server-use-recce-cloud-for-free.png){: .shadow}
 
-### 2. Sign in to Recce Cloud and retrieve your API token
+### 2. Sign in to Recce Cloud and grand access permission
+
+Once your login is successful, your local Recce will be prompted to grant access to Recce Cloud. This step is crucial for enabling the sharing functionality.
+
+![Recce API Token](../assets/images/recce-cloud/recce-cloud-connection-request-approved.png){: .shadow}
```

Comment:
The alt text "Recce API Token" should be adjusted.

Reply:
Fixed